### ISSUE TITLE
Update to v0.154.1 has broken our development environments

### ISSUE THREAD

**Issue by graememross** [microsoft/vscode-remote-release#4217]
Our vscode extensions updated yesterday from v0.154.0 to v0.154.1

Any attempt to reconnect (or rebuild) a container hosted on decker-for-windows is now throwing the following error
```
[11589 ms] Start: Run in container: test -d /vscode/vscode-server/extensionsCache && ls /vscode/vscode-server/extensionsCache || true
[11591 ms]
[11591 ms]
[11592 ms] Start: Run in container: for pid in `cd /proc && ls -d [0-9]*`; do { echo $pid ; readlink /proc/$pid/cwd ; readlink /proc/$pid/ns/mnt ; cat /proc/$pid/stat | tr "
[11631 ms] TypeError: Cannot read property 'mntNS' of undefined
        at V (c:\Users\mtltqy\.vscode\extensions\ms-vscode-remote.remote-containers-0.154.1\dist\extension.js:1:640813)
        at processTicksAndRejections (internal/process/task_queues.js:94:5)
        at async Object.t.injectIntoContainer (c:\Users\mtltqy\.vscode\extensions\ms-vscode-remote.remote-containers-0.154.1\dist\extension.js:1:637863)
        at async g (c:\Users\mtltqy\.vscode\extensions\ms-vscode-remote.remote-containers-0.154.1\dist\extension.js:1:812958)
        at async Object.t.openDockerfileDevContainer (c:\Users\mtltqy\.vscode\extensions\ms-vscode-remote.remote-containers-0.154.1\dist\extension.js:1:818279)
```
We tried rolling the update back a few versions but this failed to resolve the problem. Eventually we had to roll back to v0.132.0 before we could again connect to the container.

- VSCode Version: 1.52.1
- Local OS Version: Windows 10 Enterprise 1809
- Remote OS Version: CentOS Linux release 7.6.1810 (Core)
- Remote Extension/Connection Type: Docker

Steps to Reproduce:

1. Attempt to open the project within the container

Does this issue occur when you try this locally and all extensions are disabled?: Yes

**Comment by totycro** [comment]
I found this tweet, which mentions the error message and claims that the xargs command is somehow involved (tweet is in japanese):
https://twitter.com/rinfield/status/1339501870214127616

**Comment by chrmarti** [comment]
Could you attach your devcontainer.json, Dockerfile and the complete log from the failure (`F1` > `Remote-Containers: Show Log`)?

**Comment by totycro** [comment]
The problems seems to be related to the user. If I remove `USER www-data` from the Dockerfile, I don't get the error messsage. 
I've set `"remoteUser": "root"` in the devcontainer.json.

[Dockerfile.txt](https://github.com/microsoft/vscode-remote-release/files/5730346/Dockerfile.txt)
[devcontainer.json.txt](https://github.com/microsoft/vscode-remote-release/files/5730350/devcontainer.json.txt)
[remote-containers-log.txt](https://github.com/microsoft/vscode-remote-release/files/5730367/remote-containers-log.txt)

**Comment by chrmarti** [comment]
@totycro That reproduced the problem and I have prepared a fix for the next update, thanks.

I think setting the user in the Dockerfile or using `"runArgs"` in the devcontainer.json to the same as `"remoteUser"` should work around the problem for now.

**Comment by zombiemaker** [comment]
@chrmarti What is the actual fix?  I have a container image which sets the user to "me" and I have the remoteUser parameter in the devcontainer.json file set to "me" and I still get this message.

**Comment by chrmarti** [comment]
@zombiemaker Could you attach your devcontainer.json, Dockerfile and the complete log from the failure (`F1` > `Remote-Containers: Show Log`)?

The fix is about looking up the current process' mount namespace which fails (prior to the fix) depending on how permissions are set up.

**Comment by zombiemaker** [comment]
@chrmarti Here is the info you requested.

[Dockerfile.txt](https://github.com/microsoft/vscode-remote-release/files/5765349/Dockerfile.txt)
[devcontainer.json.txt](https://github.com/microsoft/vscode-remote-release/files/5765335/devcontainer.json.txt)
[log.txt](https://github.com/microsoft/vscode-remote-release/files/5765330/log.txt)

**Comment by chrmarti** [comment]
@zombiemaker Indeed, that's the same underlying problem and is address by the fix in code.

That fix is now available with version 0.154.2 of the extension. Thanks.
